### ISSUE TITLE
pip install doesn't create ProTECT.py

### ISSUE THREAD

**Issue by Jeltje** [BD2KGenomics/protect#50]
Following instructions at https://github.com/BD2KGenomics/protect/blob/master/MANUAL.md
`source ~/venvs/protect/bin/activate`
`pip list`
```
bd2k-python-lib (1.13.dev21)
boto (2.38.0)
cgcloud-lib (1.4a1.dev195)
dill (0.2.5)
pip (8.1.2)
protect (2.0)   <--- it exists!
PyYAML (3.11)
setuptools (12.2)
toil (3.2.0a2.dev198)
```
However, ProTECT.py is not in my path:
```
>which pip
/home/ubuntu/venvs/protect/bin/pip
>which toil
/home/ubuntu/venvs/protect/bin/toil
> which protect
>which ProTECT.py
>
```
I also cannot autocomplete pro or Pro to a tool, so nothing useful seems to be in my path.
`find /home/ubuntu/venvs/protect -iname "protect*"`
```
/home/ubuntu/venvs/protect
/home/ubuntu/venvs/protect/lib/python2.7/site-packages/protect
/home/ubuntu/venvs/protect/lib/python2.7/site-packages/protect-2.0-py2.7.egg-info
```

**Comment by arkal** [comment]
ProTECT.py is a script in protect/ . Anyway, this doesn't matter since the next PR will add `ProTECT` to the path.

**Comment by Jeltje** [comment]
I presume you mean `~/venvs/protect/lib/python2.7/site-packages/protect/`?
Actually, as the `find` command above shows (-iname makes it case insensitive), ProTECT.py is _not_ in protect. 

`ls ~/venvs/protect/lib/python2.7/site-packages/protect`
```
addons.py   binding_prediction  expression_profiling  __init__.pyc         mutation_translation.py   rankboost.py
addons.pyc  common.py           haplotyping           mutation_annotation  mutation_translation.pyc  rankboost.pyc
alignment   common.pyc          __init__.py           mutation_calling     qc
```

Let me know when the next version is ready for testing.

**Comment by arkal** [comment]
You could try cloning the above branch and running it and give me pre-merge critique

**Comment by arkal** [comment]
Resolved in #52. closing
